### PR TITLE
[Windows][CI] Fixed workaround with http_test_server

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -106,8 +106,7 @@ jobs:
         shell: bash
         run: |
           node tests/server.js &
-          # See https://github.com/php-http/client-integration-tests/issues/46 for the background of the line below.
-          php -S 127.0.0.1:10000 -t "vendor/php-http/client-integration-tests/fixture" &
+          php ./vendor/bin/http_test_server &
```
Comment (TimWolla) [inline .github/workflows/ci.yml:109]:
```suggestion
          ./vendor/bin/http_test_server &
```

**Review by TimWolla — CHANGES_REQUESTED** [review]
I see the issue now. `http_test_server` is not a PHP, but a bash script. It contains a proper shebang, so simply removing the `php` should work.

**Comment by Nyholm** [comment]
I thought I tried that.. 

Thank you

**Comment by Nyholm** [comment]
Oh. Lovely. It all is green.  =)

Thank you

File: composer.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     },
     "require-dev": {
         "ext-curl": "*",
-        "php-http/client-integration-tests": "dev-phpunit8",
+        "php-http/client-integration-tests": "dev-master",
```
Comment (GrahamCampbell) [inline composer.json:42]: `^3.0`?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (Nyholm) [reply composer.json]: Thank you

**Comment by GrahamCampbell** [comment]
PR title is a bit misleading.

**Review by GrahamCampbell — APPROVED** [review]

**Comment by Nyholm** [comment]
Thank you for the fix and reviews.

**Pull request merged**